### ISSUE TITLE
Unexpected Merge output in a specific scenario

### ISSUE THREAD

**Issue by sandkum5** [netascode/terraform-utils-nac-merge#2]
Not sure if this is an issue, but wanted to reach out/report this just in case. 

Seeing empty results for an org when the policies key is empty in one of the YAML files. 

E.g. I have 2 yaml files: ntp.yaml, ssh.yaml. 

- I have an "ntp" key defined under "policies" under ntp.yaml file. 
- If I keep the "policies" key empty under ssh.yaml file, the "default" org returns an empty result even though I have ntp variables in ntp.yaml file. 

## Not working example:

ntp.yaml
```
---
intersight:
  organization:
    default:
      policies:
        ntp:
          - name: Demo-default_ntp1
            ntp_servers: ["1.1.1.1", "1.1.1.2"]
            description: Default NTP1 Policy
            enabled: false
            timezone: "America/Los_Angeles"
    prod:
      policies:
        ntp:
          - name: Demo-prod_ntp1
            ntp_servers: ["1.1.1.3", "3.3.3.3"]
            description: Prod NTP3 Policy
            enabled: true
            timezone: "America/Los_Angeles"
```

ssh.yaml

```
---
intersight:
  organization:
    default:
      policies:
        # ssh:
    prod:
      policies:
        ssh:
          - name: prod_ssh1
            description: Prod NTP1 Policy
            enabled: true
            port: 22
            timeout: 300
            tags:
              - key: Location
                value: SAN_JOSE
              - key: ENV
                value: PROD
```

### Not Expected Output:
Sample output where the "default" key shows empty:
```
  + local_model  = {
      + intersight = {
          + organization = {
              + default = {}
              + prod    = {
                  + policies = {
                      + ntp = [
                          + {
                              + description = "Prod NTP3 Policy"
                              + enabled     = true
                              + name        = "Demo-prod_ntp1"
                              + ntp_servers = [
                                  + "1.1.1.3",
                                  + "3.3.3.3",
                                ]
                              + timezone    = "America/Los_Angeles"
                            },
                          + {
```

## Working Example:

ntp.yaml
```
---
intersight:
  organization:
    default:
      policies:
        ntp:
          - name: Demo-default_ntp1
            ntp_servers: ["1.1.1.1", "1.1.1.2"]
            description: Default NTP1 Policy
            enabled: false
            timezone: "America/Los_Angeles"
    prod:
      policies:
        ntp:
          - name: Demo-prod_ntp1
            ntp_servers: ["1.1.1.3", "3.3.3.3"]
            description: Prod NTP3 Policy
            enabled: true
            timezone: "America/Los_Angeles"
```

ssh.yaml

```
---
intersight:
  organization:
    default:
      policies:
        ssh:
    prod:
      policies:
        ssh:
          - name: prod_ssh1
            description: Prod NTP1 Policy
            enabled: true
            port: 22
            timeout: 300
            tags:
              - key: Location
                value: SAN_JOSE
              - key: ENV
                value: PROD
```

### Expected Output:
```
  + local_model  = {
      + intersight = {
          + organization = {
              + default = {
                  + policies = {
                      + ntp = [
                          + {
                              + description = "Default NTP1 Policy"
                              + enabled     = false
                              + name        = "Demo-default_ntp1"
                              + ntp_servers = [
                                  + "1.1.1.1",
                                  + "1.1.1.2",
                                ]
                              + timezone    = "America/Los_Angeles"
                            },
                          + {
                              + description = "Default NTP2 Policy"
                              + enabled     = true
                              + name        = "Demo-default_ntp2"
                              + ntp_servers = [
                                  + "2.2.2.2",
                                  + "2.2.2.1",
                                ]
                              + timezone    = "America/Los_Angeles"
                            },
                        ]
                    }
                }
              + prod    = {
                  + policies = {
                      + ntp = [
                          + {
                              + description = "Prod NTP3 Policy"
                              + enabled     = true
                              + name        = "Demo-prod_ntp1"
                              + ntp_servers = [
                                  + "1.1.1.3",
                                  + "3.3.3.3",
                                ]
                              + timezone    = "America/Los_Angeles"
                            },
                          + {
```

**Comment by danischm** [comment]
https://github.com/netascode/terraform-provider-utils/commit/8440d7db2b1a98d4d3776b37acc355407947210c

**Comment by danischm** [comment]
This should be fixed with v0.2.5 of the provider. Be aware that I am no longer maintaining this `nac-merge` module as the functionality will be integrated into another module (https://github.com/netascode/terraform-aci-nac-aci).
